### ISSUE TITLE
Leaderboard

### ISSUE THREAD

**Issue by techsionGD** [Cvolton/GMDprivateServer#745]
my leaderboard is the regular GD leaderboard, could this be something to do with megahack v6 (i have accurate leaderboard turned off) 
![image](https://user-images.githubusercontent.com/82057487/114176873-a4ab3e80-9933-11eb-905f-a11fec4d3fea.png)

**Comment by Cvolton** [comment]
I get the correct leaderboards with MegaHack v6

**Comment by EMREOYUN** [comment]
> my leaderboard is the regular GD leaderboard, could this be something to do with megahack v6 (i have accurate leaderboard turned off)
> ![image](https://user-images.githubusercontent.com/82057487/114176873-a4ab3e80-9933-11eb-905f-a11fec4d3fea.png)

So is this issue related with private server?

**Comment by techsionGD** [comment]
yes it is, its on my gdps and it shows the regular gd leaderboard

**Comment by EMREOYUN** [comment]
So you forgot change something on your clients. Please look at this link
and recreate your client:
https://github.com/Cvolton/GMDprivateServer/wiki/Creating-Windows,-Android-and-IOS-Apps

**Comment by techsionGD** [comment]
thats what i did and it still doesnt work

**Comment by EMREOYUN** [comment]
> thats what i did and it still doesnt work

Which version you tried on?

**Comment by RxNationGaming** [comment]
> my leaderboard is the regular GD leaderboard, could this be something to do with megahack v6 (i have accurate leaderboard turned off)
> ![image](https://user-images.githubusercontent.com/82057487/114176873-a4ab3e80-9933-11eb-905f-a11fec4d3fea.png)

What u use? Apk or exe?

**Comment by techsionGD** [comment]
exe

**Comment by EMREOYUN** [comment]
> exe

I requested version of Geometry Dash, not platform

**Comment by techsionGD** [comment]
I was replying to someone else.. and im on 2.1

**Comment by EMREOYUN** [comment]
> I was replying to someone else.. and im on 2.1

Sorry bout that. I remember if we do everything correcty, there should no problems in 2.1. Weird.

**Comment by Intelligent-Cat** [comment]
you also gotta replace base64-encoded links

**Comment by techsionGD** [comment]
did that too
